### PR TITLE
Fixed 'ceph daemon' commands (bsc#1181488)

### PR DESCRIPTION
**Pull request by tbazant**
Closes https://bugzilla.suse.com/show_bug.cgi?id=1181488

### CODE REVIEW

File: xml/bp_troubleshooting_logging.xml [hunk 1]
```diff
@@ -57,13 +57,13 @@
...
 
-<screen>&prompt.cephuser;ceph daemon {daemon-name} config show | less</screen>
+<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon <replaceable>DAEMON_NAME</replaceable> config show | less</screen>
 
   <para>
    For example:
   </para>
 
-<screen>&prompt.cephuser;ceph daemon osd.0 config show | less</screen>
+<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon osd.0 config show | less</screen>
```
Comment (tbazant) [inline xml/bp_troubleshooting_logging.xml:66]:
```suggestion
<screen>&prompt.cephuser;cephadm enter --name osd.0 -- ceph daemon osd.0 config show | less</screen>
```

File: xml/bp_troubleshooting_logging.xml [hunk 2]
```diff
@@ -89,7 +89,7 @@
    daemon host using ceph daemon. For example:
   </para>
 
-<screen>&prompt.cephuser;sudo ceph daemon osd.0 config set debug_osd 0/5</screen>
+<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon osd.0 config set debug_osd 0/5</screen>
```
Comment (tbazant) [inline xml/bp_troubleshooting_logging.xml:92]:
```suggestion
<screen>&prompt.cephuser;cephadm enter --name osd.0 -- ceph daemon osd.0 config set debug_osd 0/5</screen>
```

File: xml/runtime_config.xml [hunk 3]
```diff
@@ -246,11 +246,11 @@
        daemon process via a socket in <filename>/var/run/ceph</filename> from
        the node where the process is running:
       </para>
-<screen>&prompt.cephuser;ceph daemon <replaceable>DAEMON</replaceable> config set <replaceable>OPTION</replaceable> <replaceable>VALUE</replaceable></screen>
+<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon <replaceable>DAEMON</replaceable> config set <replaceable>OPTION</replaceable> <replaceable>VALUE</replaceable></screen>
       <para>
        For example:
       </para>
-<screen>&prompt.cephuser;ceph daemon osd.4 config set debug_osd 20</screen>
+<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon osd.4 config set debug_osd 20</screen>
```
Comment (tbazant) [inline xml/runtime_config.xml:253]:
```suggestion
<screen>&prompt.cephuser;cephadm enter --name osd.4 -- ceph daemon osd.4 config set debug_osd 20</screen>
```

File: xml/runtime_config.xml [hunk 4]
```diff
@@ -281,15 +281,15 @@
      You can also connect to a running daemon from the node where its process
      is running, and observe its configuration:
     </para>
-<screen>&prompt.cephuser;ceph daemon osd.0 config show</screen>
+<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon osd.0 config show</screen>
```
Comment (tbazant) [inline xml/runtime_config.xml:284]:
```suggestion
<screen>&prompt.cephuser;cephadm enter --name osd.0 -- ceph daemon osd.0 config show</screen>
```

File: xml/runtime_config.xml [hunk 5]
```diff
@@ -281,15 +281,15 @@
...
     <para>
      To view only non-default settings:
     </para>
-<screen>&prompt.cephuser;ceph daemon osd.0 config diff</screen>
+<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon osd.0 config diff</screen>
```
Comment (tbazant) [inline xml/runtime_config.xml:288]:
```suggestion
<screen>&prompt.cephuser;cephadm enter --name osd.0 -- ceph daemon osd.0 config diff</screen>
```

File: xml/runtime_config.xml [hunk 6]
```diff
@@ -281,15 +281,15 @@
...
     <para>
      To inspect a specific option:
     </para>
-<screen>&prompt.cephuser;ceph daemon osd.0 config get debug_osd</screen>
+<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon osd.0 config get debug_osd</screen>
```
Comment (tbazant) [inline xml/runtime_config.xml:292]:
```suggestion
<screen>&prompt.cephuser;cephadm enter --name osd.0 -- ceph daemon osd.0 config get debug_osd</screen>
```

**Review by tbazant — COMMENTED** [review]
Replace $ID with a specific number from examples

**Review by sebastian-philipp — CHANGES_REQUESTED** [review]
Either you use 

```xml
<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon osd.<replaceable>ID</replaceable> config set <replaceable>OPTION</replaceable> <replaceable>VALUE</replaceable></screen>
```

or

```xml
<screen>&prompt.cephuser;cephadm enter --name <replaceable>DAEMON</replaceable> -- ceph daemon <replaceable>DAEMON</replaceable> config set <replaceable>OPTION</replaceable> <replaceable>VALUE</replaceable></screen>
```

but please don't mix them:

```xml
<screen>&prompt.cephuser;cephadm enter --name osd.<replaceable>ID</replaceable> -- ceph daemon <replaceable>DAEMON</replaceable> config set <replaceable>OPTION</replaceable> <replaceable>VALUE</replaceable></screen>
```

**Review by lproven — APPROVED** [review]

**Pull request merged**